src/utils.js

```diff
@@ -88,25 +85,28 @@ const getTextButton = (userId, eventDetail) => {
   return textButton;
 };
 
-const handleButtonEvent = (
-  e,
-  textButton,
-  userId,
-  eventDetail,
-  setGoToEditEvent,
-  setEventToEdit,
-  setTextButton
-) => {
-  const textEventButton = textButton;
-
-  if (textEventButton === "edit") {
+const handleButtonEvent = async (parameters) => {
+  const {
+    e,
```

Comment:
remove this parameter, we dont need it, also remove it from the components that call this function